### PR TITLE
[SPARK]Update the Spark document Adds the insert, update and delete of the primary key table

### PR DESCRIPTION
**Pull request by hellojinsilei**
## Why are the changes needed?

Arctic Spark 0.3.2 support Insert/update/delete to primary key table，so update spark doc.

## Documentation
https://arctic.netease.com/ch/spark/spark-dml/#select-from-change-table

### CODE REVIEW

File: site/docs/ch/spark/spark-ddl.md [hunk 1]
```diff
@@ -70,11 +70,23 @@ CREATE TABLE arctic_catalog.db.sample
...
+PARTITIONED BY (pt)  
+TBLPROPERTIES (''prop1''=''val1'', ''prop2''=''val2'')
+AS SELECT ...
+```
 
 ???+danger "CREATE TABLE ... AS SELECT 在当前版本没有原子性保证"
 
 ## CREATE TABLE ... LIKE
 
+`CREATE TABLE ... LIKE` 语法会将表结构包括主键、分区以及表配置复制到新表中，但不会复制数据。
```
Comment (baiyangtx) [inline site/docs/ch/spark/spark-ddl.md:89]: 标注一下，因为 primary key 不是 spark 标准语法，所以如果源表是 arctic 表，且有主键，新建表可以复制主键这部分的schema信息，如果是其他类型的表，则无法复制

**Review by baiyangtx — CHANGES_REQUESTED** [review]

File: site/docs/ch/spark/spark-get-started.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ insert overwrite test3 values
 ( 3, "bbb", timestamp('2022-1-3 00:00:00'));
 ```
 
-> 在当前版本中, 只在无主键表上支持 insert into 语法
+> 在当前版本中, 在有主键表和无主键表上都支持 insert into 语法
```
Comment (baiyangtx) [inline site/docs/ch/spark/spark-get-started.md:77]: insert or upsert 的参数需要说明一下。

**Review by baiyangtx — CHANGES_REQUESTED** [review]

File: site/docs/ch/spark/spark-get-started.md (hunk 2 above)
Comment (zhoujinsong) [reply site/docs/ch/spark/spark-get-started.md:77]: Maybe you should remove this note if all kind of table are  supported now.

File: site/docs/ch/spark/spark-ddl.md [hunk 3]
```diff
@@ -70,18 +70,31 @@ CREATE TABLE arctic_catalog.db.sample
...
 
 ``` 
 CREATE TABLE arctic_catalog.db.sample
 LIKE arctic_catalog.db.sample2
 USING arctic
 ```
 
+???+note "因为 primary key 不是 spark 标准语法，所以如果源表是 arctic 表，且有主键，新建表可以复制主键这部分的schema信息，如果是其他类型的表，则无法复制"
```
Comment (zhoujinsong) [inline site/docs/ch/spark/spark-ddl.md:97]: `Spark` and `Arctic` are proper nouns so we set the first character to upper case.
A white space should arround the English word in Chinese document.

**Comment by zhoujinsong** [comment]
@hellojinsilei Could you please rename this PR's name to describe what you do in more detail ？

**Review by zhoujinsong — APPROVED** [review]
LGTM.

**Review by baiyangtx — APPROVED** [review]

**Pull request merged**